rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -211,6 +212,18 @@ pub async fn propose_to_set_network_economics_and_wait(
 }
 
 pub type DeployedSnsStartingInfo = BTreeMap<SnsCanisterType, (ProposalInfo, SnsWasm)>;
+pub type SnsWasms = BTreeMap<SnsCanisterType, SnsWasm>;
```

Comment:
Question for reviewer: Did a type like this exist already somewhere?

Reply:
Yes, e.g., 

```rust
impl Version {
    /// Get the new hashes from next_version as a list of (SnsCanisterType, wasm_hash)
    pub(crate) fn changes_against(
        &self,
        next_version: &Self,
    ) -> Vec<(SnsCanisterType, Vec<u8> /*wasm hash*/)> {
    ...
    }
```